kind/kind-local-up.sh

```diff
@@ -14,44 +34,41 @@ nodes:
 - role: worker
 EOF
 
-cat << EOF > kind-conf-ipv6.yaml
+    cat << EOF > cilium-conf.yaml
 kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+- role: worker
+- role: worker
+- role: worker
 networking:
   disableDefaultCNI: true
-  ipFamily: ipv6
 EOF
 
-#cluster=nocni
-#conf="calico-conf.yaml"
-
-#cluster=cipv6
-#conf=kind-conf-ipv6.yaml
-
-## calico conf == no cni, so use it for antrea/calico/whatever
-#cluster=antrea
-#conf=calico-conf.yaml
-
-cluster=calico
-conf=calico-conf.yaml
+    cat << EOF > kind-conf-ipv6.yaml
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+networking:
+  disableDefaultCNI: true
+  ipFamily: ipv6
+EOF
+}
 
 function install_k8s() {
-    if kind delete cluster --name=${cluster}; then
+    if kind delete cluster --name=${CLUSTER}; then
     	echo "deleted old kind cluster, creating a new one..."
-    fi	    
-    kind create cluster --name=${cluster} --config=${conf} 
-    export KUBECONFIG="$(kind get kubeconfig-path --name=kind-${cluster})"
-    chmod 755 ~/.kube/kind-config-kind
-    export KUBECONFIG="$(kind get kubeconfig-path --name=kind-${cluster})"
+    fi
+    kind create cluster --name=${CLUSTER} --config=${CONFIG}
     until kubectl cluster-info;  do
-        echo "`date`waiting for cluster..."
+        echo "`date` waiting for cluster..."
         sleep 2
     done
 }
 
 function install_calico() {
     kubectl get pods
-    kubectl apply -f ./calico3.12.3.yaml
+    kubectl apply -f ./calico312.yaml
```

Comment:
Not sure which is the correct one here.

Reply:
you can replace this however you want.. if it works, it works :)